react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Carousel extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      openId: 0
+    };
+    this.counting();
+  }
+
+  counting() {
+    this.timerID = setInterval(
+      () => this.tick(), 3000);
+  }
+
+  tick() {
+    if (this.state.openId === 4) {
```

Comment:
What if there are more than 5 image in the array? How can we dynamically figure out the last index in the array?